src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -2702,19 +2708,6 @@ public boolean onBlockPlace(BlockPlaceEvent e, ItemStack item) {
 				}
 				else return false;
 			}
-		}, new BlockBreakHandler() {

```

Comment:
Why is this part of the code removed?

Reply:
Because it’s handled by the Containment Pickaxe handler like the Broken_Spawner is, didn’t see any reason to have redundant code for setting the type when it could be handled in one spot.